libraries

```diff
@@ -0,0 +1,2 @@
+./containers/containers.agda-lib
```

Comment:
This should not be added yet, right?

Reply:
😱 Yes, this file should not be added, sorry.